e2e-tests/tests/enketo.spec.js

```diff
@@ -251,20 +251,21 @@ test.describe('Enketo', () => {
     await expect(frame.getByRole('heading', { name: 'Successful' })).toBeVisible();
   });
 
-  test('default value is consistent between rendering enketo directly or via iframe', async ({ page }) => {
+  test('default value is consistent between rendering enketo directly and via iframe', async ({ page }) => {
     await login(page);
 
     const queryWithSpaces = '?d[/data/first_name]=hello earth + hello mars %20 hello jupiter %2B hello saturn';
 
     await page.goto(`${appUrl}/enketo-passthrough/${publishedForm.enketoId}${queryWithSpaces}`);
     await expect(page.getByRole('heading', { name: publishedForm.name })).toBeVisible();
 
-    const defaultValueInEnketo = await page.getByLabel('First Name').inputValue();
+    await expect(page.getByLabel('First Name')).toHaveValue('hello earth + hello mars   hello jupiter + hello saturn');
```

Comment:
(No change needed, just a note.) I was initially confused by the first `+` here. I thought, shouldn't the `+` in `queryWithSpaces` be converted to a space? But then I realized that this is going directly to /enketo-passthrough without involving Frontend. Enketo is unchanged and will still treat `+` as a plus, not a space. What's changed with this PR is that Frontend will never pass `+` to Enketo. Frontend will pass `%20` for a space or `%2B` for plus, but it won't pass `+`. A `+` passed to Frontend will end up getting passed to Enketo as `%20`. We can sort of see that happening in the assertion below.